subprojects/provider-plugins/src/main/resources/META-INF/gradle-plugins/org.gradle.kotlin.kotlin-dsl-project.properties

```diff
@@ -0,0 +1 @@
+implementation-class=org.gradle.kotlin.dsl.provider.plugins.KotlinScriptBasePlugin
```

Comment:
❓ Does this mean we'll get plugin spec accessors for these plugins too?

Reply:
Nope, only the gradle jars are considered when generating the plugin id extensions.